sources/packages/backend/workflow/test/2023-2024/parttime-assessment/awards-amounts/parttime-assessment-2023-2024-awards-amount-CSGD.e2e-spec.ts

```diff
@@ -0,0 +1,226 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+import {
+  DependentEligibility,
+  createFakeStudentDependentEligible,
+} from "../../../test-utils/factories";
+import { YesNoOptions } from "@sims/test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-awards-amount-CSGD for 3 or more dependants.`, () => {
+  it("Should determine federalAwardCSGDAmount when calculatedDataTotalFamilyIncome <= limitAwardCSGDIncomeCap", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataCRAReportedIncome = 20000;
+    assessmentConsolidatedData.studentDataRelationshipStatus = "married";
+    assessmentConsolidatedData.studentDataIsYourSpouseACanadianCitizen =
+      YesNoOptions.Yes;
+    assessmentConsolidatedData.partner1CRAReportedIncome = 20000;
+    assessmentConsolidatedData.studentDataDependants = [
+      createFakeStudentDependentEligible(
+        DependentEligibility.Eligible0To18YearsOld,
```

Comment:
Shouldn't there be 3 or more dependents?

Reply:
Thanks for spotting this. It is now including 3 dependents in the latest commit. Please a have look.